Automattic/wp-calypso
# state(invites): remove `invites-sent` flux store

#### Changes proposed in this Pull Request

* Remove the `invites-sent` flux store and replace actions with internal component methods

#### Testing instructions

For some parts of the testing instructions it makes sense to run this branch locally because you have to make changes in code to test a certain error scenario.

To be able to test error responses from the API apply the following diff so you can submit invites despite invalid input.

```diff
diff --git a/client/my-sites/people/invite-people/index.jsx b/client/my-sites/people/invite-people/index.jsx
index f0eaa3f853..acf5e3763e 100644
--- a/client/my-sites/people/invite-people/index.jsx
+++ b/client/my-sites/people/invite-people/index.jsx
@@ -323,6 +323,7 @@ class InvitePeople extends React.Component {
        };
 
        isSubmitDisabled = () => {
+               return false;
                const { success, usernamesOrEmails } = this.state;
                const invitees = Array.isArray( usernamesOrEmails ) ? usernamesOrEmails : [];
```

1. Open this branch on `/people/new/:site`
2. Add valid and invalid usernames and/or emails (test all 3 scenarios: all valid, all invalid and mixed)
3. Hit submit and make sure that valid entries are removed from the form and invalid entries are still there
4. For the last scenario block the request URL via browser devtools, is an error notice shown?

part of #24149
